package-res/resources/web/pentaho/data/filter/AbstractTreeFilter.js

```diff
@@ -25,12 +25,56 @@ define([
    * @name AbstractTreeFilter
    * @memberOf pentaho.data.filter
    * @class
+   * @extends pentaho.data.filter.AbstractFilter
    * @abstract
    * @amd pentaho/data/filter/AbstractTreeFilter
    *
    * @classdesc The `AbstractTreeFilter` class is the abstract base class of
    * classes that represent a filter.
    *
+   * @example
+   * <caption> Create a new class <code>DerivedAbstractTreeFilter</code> what to put here???</caption>
```

Comment:
A developer that needs to derive a class from `AbstractTreeFilter` would probably prefer to look at the code.
If we are to provide examples, they should be on how the filters are used.

If it is indeed useful to provide an example on deriving a class from `AbstractTreeFilter, why not use a concrete example such as an XOR, NAND?

This comment applies to most example blocks.